src/textField/TextField.tsx

```diff
@@ -272,6 +275,10 @@ export class TextField extends PureComponent<TextFieldProps, TextFieldState> {
       attrs['aria-invalid'] = 'true'
     }
 
+    if (dataEncryptedName) {
+      attrs['data-encrypted-name'] = dataEncryptedName
```

Comment:
I don't see any value in passing this prop instead of adding the `data-ecrypted-name` on the component itself, is there a strong reason for this verbbose?

Reply:
Every prop is using camelCase, so I just respected that format 🙂